int32 roboyID
geometry_msgs/Vector3[] viaPoints